tests/GenericExtractorJobTest.php

```diff
@@ -331,7 +331,7 @@ protected function getJob(JobConfig $config, array $attributes, array $metadata)
      * @param $field
      * @param $expectedValue
      */
-    public function testGetPlaceholder($field, $expectedValue)
+    public function testGetPlaceholder($field, $expectedValue): void
```

Comment:
tady u většiny testy chybí typehint u parametrů

Reply:
pozeram, ze to som zabudol prejst, vdaka idem to pridat.